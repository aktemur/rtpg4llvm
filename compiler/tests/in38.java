/*compunit*/
extern int* x;
extern int x[];
extern int *x[];