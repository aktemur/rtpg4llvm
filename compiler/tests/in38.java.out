mkProgram(3, new ExternDecl(new PtrType(new IntType()), "x"),
   new ExternDecl(new ArrayType(new IntType()), "x"),
   new ExternDecl(new ArrayType(new PtrType(new IntType())), "x"))
